website/src/parsers/js/hermes.js

```diff
@@ -0,0 +1,44 @@
+import defaultParserInterface from './utils/defaultESTreeParserInterface';
+import HermesWorkerClient from './hermes/HermesWorkerClient';
+
+const ID = 'hermes';
+export const defaultOptions = {
+  jsx: true,
+};
+export const parserSettingsConfiguration = {
+  fields: [
+    'jsx',
+  ],
+};
+
+export default {
+  ...defaultParserInterface,
+
+  id: ID,
+  displayName: ID,
+  version: 'N/A',
+  homepage: 'https://hermesengine.dev/',
+  locationProps: new Set(['loc', 'range']),
+
+  loadParser(callback) {
+    callback(new HermesWorkerClient());
+  },
+
+  async parse(hermes, code, options) {
+    const hermesOutput = await hermes.run(['-dump-ast', '--pretty-json', '-dump-source-location', options.jsx ? '-parse-jsx' : ''].filter(Boolean), code);
+    try {
+      return JSON.parse(hermesOutput);
+    } catch (e) {
+      console.error(e);
```

Comment:
Console leftover? I don't think we do this in other plugins.

Reply:
Yeah I can probably delete this. See rationale for having it in the first place in the comment below.